### ISSUE TITLE
Update Cloud API to use existing configurations instead of creating own

### ISSUE THREAD

**Issue by liza-mae** [elastic/elastic-stack-testing#503]
Recently this command started failing on CI, ESTF cloud run 43: 

```
POST 
https://admin.staging.foundit.no/api/v0.1/v1-regions/us-east-1/platform/configuration/instances
Response: {"ok":false,"message":"There was an internal server error."}
```
From slack with @imkarrer: they are deprecating the clusters api completely and will have  a new deployments api which provides a cleaner interface.

I will file an issue in the Java SDK to support the new deployments api and also discuss with @vogxn  if it makes sense for me to move to Go client (which is kept more up to date) or maybe we can get the Java client on a more frequent update schedule. 

@imkarrer we should make sure we can get these updates before the clusters API is completely removed, do we have a schedule on that?  Is there documentation on the new deployment API?

In the meantime I will make an update to use an existing configuration for kibana, ml and elasticsearch to fix the internal server error.

**Comment by vogxn** [comment]
A little bit confused on why the `/platform/configuration/instances` error was related to the deprecation of `/clusters`. Are these two separate issues? 

There is no hard timeline on deprecating `/clusters` yet so it will need to be functional for a good while. For now, I think we can quite easily spin up a fresh release of the Java SDK to include the deployments endpoints. This will ensure that your code has time to transition to using the new endpoints.

**Comment by liza-mae** [comment]
@vogxn yes two issues - it started with the failure on platform/configuration/instances which I am now using existing configurations and from my conversation with Isaac about this he found out that the clusters API was being deprecated too, so I wanted to open an issue to discuss support for the java sdk and/or if we should move to go sdk. 

Getting a new Java SDK with deployment endpoints would be great.
